src/hooks/useContacts/index.test.jsx

```diff
@@ -66,49 +64,43 @@ describe("usePeople", () => {
 
     afterEach(() => {
       cache.clear();
+      jest.clearAllMocks();
+      cleanup();
```

Comment:
I think https://testing-library.com/docs/react-testing-library/api/#cleanup is added automatically? Sure you need it here?